addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -715,25 +740,10 @@ registerModel({
             if (this._fallBackTimeouts[token]) {
                 return;
             }
-            this._fallBackTimeouts[token] = browser.setTimeout(async () => {
-                delete this._fallBackTimeouts[token];
-                const peerConnection = this._peerConnections[token];
-                if (!peerConnection || !this.channel) {
-                    return;
-                }
-                if (this._outGoingCallTokens.has(token)) {
-                    return;
-                }
-                if (peerConnection.iceConnectionState === 'connected') {
-                    return;
-                }
-                this._addLogEntry(token, `calling back to recover ${peerConnection.iceConnectionState} connection, reason: ${reason}`);
-                await this._notifyPeers([token], {
-                    event: 'disconnect',
-                });
-                this._removePeer(token);
-                this._callPeer(token);
-            }, delay);
+            this._fallBackTimeouts[token] = browser.setTimeout(
+                this._onRecoverConnectionTimeout.bind(this, token, reason),
```

Comment:
ok for now, but having to bind such params highlights that the method is probably not on the right record (and the right record doesn't even exist yet)